cmd/agent/subcommands/run/command.go

```diff
@@ -50,6 +50,8 @@ import (
 	logsAgent "github.com/DataDog/datadog-agent/comp/logs/agent"
 	"github.com/DataDog/datadog-agent/comp/metadata"
 	"github.com/DataDog/datadog-agent/comp/metadata/runner"
+	"github.com/DataDog/datadog-agent/comp/netflow"
+	"github.com/DataDog/datadog-agent/comp/netflow/server"
```

Comment:
```suggestion
	netflowServer "github.com/DataDog/datadog-agent/comp/netflow/server"
```

I think it can bring more clarity since there are multiple "server" in this file.

Reply:
Fixed.